view/next-project/src/components/purchasereports/DeleteModal.tsx

```diff
@@ -4,7 +4,7 @@ import React, { Dispatch, FC, SetStateAction } from 'react';
 import { CloseButton, Modal, OutlinePrimaryButton, PrimaryButton } from '../common';
 import { del } from '@api/api_methods';
 
-interface ModalProps {
+export interface ModalProps {
```

Comment:
```suggestion
interface ModalProps {
```
Propsをstorybookで使ってないなら、exportしなくてもいいかもです！